packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5466,6 +5466,485 @@ curl -v -X DELETE \
+"https://${yourOktaDomain}/api/v1/apps/${applicationId}/connections/default"
+```
+
+#### Response example
+
+```json
+{
+    "authScheme": "TOKEN",
+    "status": "ENABLED",
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/apps/${applicationId}/connections/default",
+            "hints": {
+                "allow": [
+                    "POST",
+                    "GET"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://${yourOktaDomain}/api/v1/apps/${applicationId}/connections/default/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Set default Provisioning Connection for application
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="post" url="/api/v1/apps/${applicationId}/connections/default" />
+
+Sets the default Provisioning Connection for an application.
+
+#### Request parameters
+
+| Parameter       | Description                                | Parameter Type   | DataType   | Required | Default |
+| :-------------- | :----------------------------------------- | :--------------- | :--------- | :------- | :-------|
+| activate        | Activate the provisioning connection       | Query            | Boolean    | FALSE    | FALSE   |
+| applicationId   | `id` of an [app](#application-object)      | URL              | String     | TRUE     |         |
+| profile   | Provisioning profile     | Body              | [Provisioning Connection Profile](#provisioning-connection-profile-object)   | TRUE     |         |
+
+#### Response parameters
+
+The new default [Provisioning Connection](#provisioning-connection-object).
+
+#### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+    "profile": {
+        "authScheme": "TOKEN",
+        "token": "TEST"
+    }
+}' "https://${yourOktaDomain}/api/v1/apps/${applicationId}/connections/default?activate=TRUE"
+```
+
+#### Response example
+
+```json
+{
+    "authScheme": "TOKEN",
+    "status": "ENABLED",
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/apps/${applicationId}/connections/default",
+            "hints": {
+                "allow": [
+                    "POST",
+                    "GET"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://${yourOktaDomain}/api/v1/apps/${applicationId}/connections/default/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Activate default Provisioning Connection for application
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="post" url="/api/v1/apps/${applicationId}/connections/lifecycle/activate" />
```

Comment:
Should it be '/connections/default/lifecycle/' instead of '/connections/lifecycle/' ?

Reply:
yes you are right, nice catch